syntax = "proto3";

package response_models;
import "vector2d.proto";

message WeaponDirectionResponseModel {
	int32 player_id = 1;
	util_models.Vector2D direction = 2;
}
